tests/test_sensors.py

```diff
@@ -230,6 +231,44 @@ def test_sensors(
         ) < 9.0e-2 * np.linalg.norm(gt.astype(float)), f"Incorrect {sensor_type} output"
 
 
+@pytest.mark.parametrize(
+    "scene_and_dataset, sensor_type",
+    list(itertools.product(_semantic_scenes, all_base_sensor_types))
+    + list(itertools.product(_non_semantic_scenes, all_base_sensor_types[0:2])),
+)
+def test_duplicate_sensors(
+    scene_and_dataset,
+    sensor_type,
+    make_cfg_settings,
+):
+    scene = scene_and_dataset[0]
+    if not osp.exists(scene):
+        pytest.skip("Skipping {}".format(scene))
+    scene_dataset_config = scene_and_dataset[1]
+
+    make_cfg_settings[sensor_type] = True
+    make_cfg_settings["scene"] = scene
+    make_cfg_settings["scene_dataset_config_file"] = scene_dataset_config
+
+    cfg = make_cfg(make_cfg_settings)
+    agent_id = 0
+    additional_sensors = cfg.agents[agent_id].sensor_specifications
+
+    with habitat_sim.Simulator(cfg) as sim:
+        for sensor_spec in additional_sensors:
+            try:
+                sim.add_sensor(sensor_spec)
+            except Exception:
```

Comment:
BareExceptions are bad here, especially for test. Atleast do a runtime error or something?